tests/test_linestring.py

```diff
@@ -69,10 +70,12 @@ def test_from_generator():
 def test_from_empty():
     line = LineString()
     assert line.is_empty
+    assert isinstance(line.coords, CoordinateSequence)
     assert line.coords[:] == []
 
     line = LineString([])
     assert line.is_empty
+    assert isinstance(line.coords, CoordinateSequence)
     assert line.coords[:] == []
```

Comment:
Could you also add a test for doing `np.array(line.coords)` for an empty LineString? 
You can add it to one of the tests below that already require numpy (and the same for Point and LinearRing)

Reply:
No problem, done. Also corrected an issue with a test class name and a test constructing a LineString when it should have been a LinearRing.